docs/tutorials/scilla-tutorials/nft-collection/minting-token.md

```diff
@@ -0,0 +1,51 @@
+---
+sidebar_position: 5
+---
+
+# Minting a token
+
+With the metadata available and a decision made on if the project is using ```token_uri``` or ```base_uri``` we can begin the final step, minting a token.
+
+## base_uri
+
+As discussed previously, the base_uri mode offloads the handling of images and metadata to the API and as such, the minter does not have to provide a token_uri as that is calculated by base_uri/token_id. If a token_uri is provided here, then base_uri is overridden to show the token_uri for this particular token.
+
+```base_uri``` is suited best to randomised minting projects which do not want to expose potential data directly on the chain. This can have other implications regarding failed transactions inadvertently revealing the data for a token depending on the implementation.
```

Comment:
I think you are completely misunderstood the base URI.
base URI is not applicable for randomized minting and it's not about exposing potential data directly on the chain.